### PR TITLE
chore: Fix style issues

### PR DESCRIPTION
**Pull request by erezrokah**
🎉 Thank you for making CloudQuery awesome by submitting a PR 🎉

#### Summary

Same as https://github.com/cloudquery/cq-provider-okta/pull/76 for `digitalocean`.

The issues in this PR are fixed based on the results of `golangci-lint run --config ../.github/misc/providers/.golangci.yml` (golint file from [PR](https://github.com/cloudquery/.github/pull/107))

---

Use the following steps to ensure your PR is ready to be reviewed

- [x] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [x] Run `go fmt` to format your code 🖊
- [x] Lint your changes via `golangci-lint run --new-from-rev main` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [x] Update or add tests. Learn more about testing [here](https://docs.cloudquery.io/docs/developers/sdk/testing) 🧪
- [x] Update the docs by running `go run ./docs/docs.go` and committing the changes 📃
- [x] Ensure the status checks below are successful ✅

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -31,15 +27,30 @@ type Client struct {
...
-func (c *Client) Logger() hclog.Logger {
-	return c.logger
-}
-
 type DoCredentialStruct struct {
 	Api    bool
 	Spaces bool
 }
 
+type SpacesCredentialsProvider struct {
```
Comment (erezrokah) [inline client/client.go:35]: Declaration order

File: client/client.go [hunk 2]
```diff
@@ -134,9 +138,9 @@ func getTokenFromEnv() string {
 	return ""
 }
 
-func getSpacesTokenFromEnv() (string, string) {
-	spacesAccessKey := os.Getenv("SPACES_ACCESS_KEY_ID")
-	spacesSecretKey := os.Getenv("SPACES_SECRET_ACCESS_KEY")
+func getSpacesTokenFromEnv() (spacesAccessKey string, spacesSecretKey string) {
```
Comment (erezrokah) [inline client/client.go:141]: `confusing-results: unnamed results of the same type may be confusing, consider using named results`

File: client/config.go [hunk 3]
```diff
@@ -14,7 +14,7 @@ type Config struct {
 	SpacesDebugLogging bool `hcl:"spaces_debug_logging,optional"`
 }
 
-func (c Config) Example() string {
+func (Config) Example() string {
```
Comment (erezrokah) [inline client/config.go:17]: Unused receiver

File: resources/droplets.go [hunk 4]
```diff
@@ -8,6 +8,11 @@ import (
 	"github.com/digitalocean/godo"
 )
 
+type neighborWrapper struct {
```
Comment (erezrokah) [inline resources/droplets.go:11]: Declaration order

File: resources/firewalls.go [hunk 5]
```diff
@@ -8,6 +8,11 @@ import (
 	"github.com/digitalocean/godo"
 )
 
+type firewallDroplets struct {
```
Comment (erezrokah) [inline resources/firewalls.go:11]: Declaration order

File: resources/floating_ips.go [hunk 6]
```diff
@@ -8,6 +8,11 @@ import (
 	"github.com/digitalocean/godo"
 )
 
+type floatingIpWrapper struct {
```
Comment (erezrokah) [inline resources/floating_ips.go:11]: Declaration order

File: resources/load_balancers.go [hunk 7]
```diff
@@ -8,6 +8,11 @@ import (
 	"github.com/digitalocean/godo"
 )
 
+type lbDroplets struct {
```
Comment (erezrokah) [inline resources/load_balancers.go:11]: Declaration order

File: resources/spaces.go [hunk 8]
```diff
@@ -11,6 +11,15 @@ import (
 	"github.com/pkg/errors"
 )
 
+type WrappedBucket struct {
```
Comment (erezrokah) [inline resources/spaces.go:14]: Declaration order

File: resources/spaces.go [hunk 9]
```diff
@@ -157,9 +166,9 @@ func fetchSpaces(ctx context.Context, meta schema.ClientMeta, parent *schema.Res
 		if !svc.CredentialStatus.Spaces {
 			log.Warn("Spaces credentials not set. skipping")
 			return nil
-		} else {
-			return err
 		}
+
+		return err
```
Comment (erezrokah) [inline resources/spaces.go:171]: Unnecessary indentation of `else return`

File: resources/spaces.go [hunk 10]
```diff
@@ -222,23 +231,14 @@ func fetchSpaceCorsRules(ctx context.Context, meta schema.ClientMeta, parent *sc
 	var ae smithy.APIError
 	r := parent.Item.(*WrappedBucket)
 	svc := meta.(*client.Client).S3
-	CORSOutput, err := svc.GetBucketCors(ctx, &s3.GetBucketCorsInput{Bucket: r.Name}, func(options *s3.Options) {
+	corsOutput, err := svc.GetBucketCors(ctx, &s3.GetBucketCorsInput{Bucket: r.Name}, func(options *s3.Options) {
```
Comment (erezrokah) [inline resources/spaces.go:234]: Var naming

File: resources/volumes.go [hunk 11]
```diff
@@ -8,6 +8,11 @@ import (
 	"github.com/digitalocean/godo"
 )
 
+type volumeDroplets struct {
```
Comment (erezrokah) [inline resources/volumes.go:11]: Declaration order

File: client/client.go (hunk 2 above)
Comment (disq) [reply client/client.go:141]: I don't think we need named results in this instance.

```suggestion
func getSpacesTokenFromEnv() (string, string) {
```

**Review by disq — COMMENTED** [review]
Maybe `cq-gen` should generate source code in correct declaration order.

**Comment by erezrokah** [comment]
> Maybe `cq-gen` should generate source code in correct declaration order.

Good point. Opened https://github.com/cloudquery/cq-gen/issues/99 to track this, and also disable the `confusing-results` rule

**Review by disq — APPROVED** [review]

**Pull request merged**